data/helpers.d/nodejs

```diff
@@ -125,7 +125,10 @@ ynh_install_nodejs () {
     test -x /usr/bin/npm && mv /usr/bin/npm /usr/bin/npm_n
 
     # If n is not previously setup, install it
-    if ! test $(n --version > /dev/null 2>&1)
+    if ! type $(n --version > /dev/null 2>&1)
```

Comment:
Not sure to understand why change `test` to `type` ? Note that if `n` is not installed, I'm guessing `$(n --version)` will return empty string, but `type` (with no arg) returns 0 and `test` (with no arg) returns 1

So it looks to me like the behavior changes in a unexpected way but maybe I'm missing something

Reply:
damn! this is a typo